### DISCUSSION TITLE
How can I create a multi-square?

### DISCUSSION THREAD

**Discussion by soasada** [terrastruct/d2#651] [Q&A] [answered]
Something along these lines: 

<img width="215" alt="image" src="https://user-images.githubusercontent.com/7634531/212065414-53eeb43c-fb69-45c1-ad1f-19108ee8b8a0.png">

Is it possible?

**Comment by alixander** [comment, marked as answer]
<img width="462" alt="Screen Shot 2023-01-12 at 6 47 16 AM" src="https://user-images.githubusercontent.com/3120367/212097588-a8a4f8de-8f52-4f29-99b0-f5695bfb2c18.png">

https://play.d2lang.com/?script=yixRyCxWKMgvLs5Mykm1UqjmUlAAG6sH86aVQklRaSpXLRcgAAD__w%3D%3D&
